users/users/keycloak/schemas.py

```diff
@@ -326,20 +331,31 @@ class RefreshTokenRequest(OAuthRequest):
     refresh_token: str
 
 
-class ClientViewsUsersAccessToken(object):
-    @classmethod
-    async def create_instance(cls):
-        instance = ClientViewsUsersAccessToken()
-        instance.token = ""
-        await instance.request_new_token()
-        return instance
+class AdminAccessToken:
+    """
+    This class provides simple caching access token to 
+    reduce count of calls to KeyCloak
+    Usage:
+        admin_access_token = AdminAccessToken()
+        # read token from cache or create new one
+        token = await admin_access_token.token()
+        # In case of token expired, just request new one ...
+        admin_access_token.request_new_token()
+        token = await admin_access_token.token()
+        # .. and retry request
+
+    TODO: think about to add refresh by refresh token
+    """
+
+    def __init__(self) -> None:
+        self.__token = None
+
+    async def token(self):
+        if self.__token is None:
+            await self.request_new_token()
+        return self.__token
```

Comment:
If empty token (string) is not suitable as well, then we could
```suggestion
        return self.__token or await self.request_new_token()
```